setup.py

```diff
@@ -7,6 +7,7 @@
 
 extras_require = {
     'test': [
+        "hypothesis>=4.18.0,<5",
         "pytest>=3.6.0",
         "pytest-xdist",
```

Comment:
Just a note. The newest `pytest-xdist` version `1.28.0` requires `pytest>=4.4.0`. It's fine here since pytest isn't constrained like `"pytest>=3.6.0,<4"`, and `pytest` version `4.4.0` gets installed, but it might be worth updating explicitly